app/controllers/api/motions_controller.rb

```diff
@@ -24,6 +24,12 @@ def update_outcome
     respond_with_resource
   end
 
+  def closed
+    load_and_authorize :group
+    @motions = @group.closed_motions.order(closed_at: :desc)
```

Comment:
alright, so `instantiate_collection` can take a block, which will modify the `visible_records` in whatever way you'd like. So, for example

```
def index
  instantiate_collection { |collection| collection.where(private: true) }
end

def visible_records
  Discussion.all
end
```

Will return all private discussions (it's magic, but not too magic)

So, here, we want something like

```
def index
  load_and_authorize :discussion
  instantiate_collection { |collection| collection.where(discussion: @discussion).order(:created_at) }
  respond_with_collection
end

def closed
  load_and_authorize :group
  instantiate_collection { |collection| collection.where(discussion_id: @group.discussion_ids).order(closed_at: :desc) }
end

def visible_records
  Queries::VisibleMotions.new(user: current_user, groups: current_user.groups)
end
```